### ISSUE TITLE
Question on fine-tuning steps of the pretrained model

### ISSUE THREAD

**Issue by xiangdal** [cbfinn/maml#21]
Hi Chelsea,

In Figure 3, when compared with pretrained models, MAML demonstrates superior effectiveness on fast learning with a few gradient steps. I am wondering, it is also worthwhile to compare with the pretrained model when it is more thoroughly fine-tuned, say after 100 fine-tuning steps during meta-testing?

Thank you!

**Comment by cbfinn** [comment]
Yes, that would be good to compare to. Though, I don't expect to see too much of a performance increase from using more fine-tuning steps and a smaller step size, given that the amount of data is fixed and relatively small. [The gradient steps are batch gradient steps, not minibatch stochastic gradient steps.]

In Figure 4 of [this paper](https://arxiv.org/abs/1710.11622), we did run a network initialized from scratch and a MAML-learned model, both for 100 steps of gradient descent at test time, and observed that performance of the randomly-initialized network saturated fairly quickly.
